zipline/src/jvmMain/CMakeLists.txt

```diff
@@ -5,7 +5,6 @@ set(CMAKE_C_STANDARD 99)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
-set(CMAKE_CXX_COMPILER "clang++")
```

Comment:
Not using clang on Linux because I couldn’t figure out how to build with it on Linux without dragging in newer libstdc++.

I believe we should still get clang on macOS via some defaults? If not we could add it to `release.yml`.

Reply:
Confirmed.
```
-- The C compiler identification is AppleClang 13.0.0.13000029
-- The CXX compiler identification is AppleClang 13.0.0.13000029
```